Sources/Experiments/ExPlat.swift

```diff
@@ -30,6 +30,26 @@ import Cocoa
         UserDefaults.standard.object(forKey: enrolledKey) != nil
     }
 
+    // MARK: - Configuration Properties
+
+    var platform: String {
+        return service.platform
+    }
+
+    var oAuthToken: String? {
+        return service.oAuthToken
+    }
+
+    var userAgent: String? {
+        return service.userAgent
+    }
+
+    var anonId: String? {
+        return service.anonId
+    }
```

Comment:
I assume these properties are added for the new test cases to make assertions? Considering these new properties are not used any library code, IMO, it would be more robust to use `OHHTTPStubs` to intercept HTTP requests and assert that the requests contain appropriate values. What do you think?